src/machine/board_btt_skr_pico.go

```diff
@@ -0,0 +1,121 @@
+//go:build btt_skr_pico
+
+// This contains the pin mappings for the BigTreeTech SKR Pico.
+//
+// Purchase link: https://biqu.equipment/products/btt-skr-pico-v1-0
+// Board schematic: https://github.com/bigtreetech/SKR-Pico/blob/master/Hardware/BTT%20SKR%20Pico%20V1.0-SCH.pdf
+// Pin diagram: https://github.com/bigtreetech/SKR-Pico/blob/master/Hardware/BTT%20SKR%20Pico%20V1.0-PIN.pdf
+
+package machine
+
+// TMC stepper driver motor direction.
+// X/Y/Z/E refers to motors for X/Y/Z and the extruder.
+const (
+	X_DIR = GPIO10
+	Y_DIR = GPIO5
+	Z_DIR = ADC2
```

Comment:
Are these used as GPIO? If so prefer the GP designation over ADC

Reply:
@EllaFoxo are you planning to make a change here based on @soypat comment?

Reply:
> @EllaFoxo are you planning to make a change here based on @soypat comment?

I will yeah. Just haven't had a lot of time on my hands lately. Got more time at the start of June, so you will likely see it done then